engine/src/main/scala/cromwell/engine/workflow/WorkflowManagerActor.scala

```diff
@@ -94,8 +98,10 @@ class WorkflowManagerActor(config: Config,
            jobStoreActor: ActorRef,
            callCacheReadActor: ActorRef,
            jobTokenDispenserActor: ActorRef,
-           backendSingletonCollection: BackendSingletonCollection) = this(
-    ConfigFactory.load, workflowStore, serviceRegistryActor, workflowLogCopyRouter, jobStoreActor, callCacheReadActor, jobTokenDispenserActor, backendSingletonCollection)
+           backendSingletonCollection: BackendSingletonCollection,
+           abortJobsOnTerminate: Boolean) = this(
+    ConfigFactory.load, workflowStore, serviceRegistryActor, workflowLogCopyRouter, jobStoreActor, callCacheReadActor,
```

Comment:
ToL: Since this entire list is triplicated, can we DRY it at all?

(e.g. wrap it up in a `WmaParameters` case class?)